activerecord/lib/active_record/connection_adapters/postgresql/column.rb

```diff
@@ -2,18 +2,8 @@ module ActiveRecord
   module ConnectionAdapters
     # PostgreSQL-specific extensions to column definitions in a table.
     class PostgreSQLColumn < Column #:nodoc:
-      attr_accessor :array
-
-      def initialize(name, default, cast_type, sql_type = nil, null = true, default_function = nil)
-        if sql_type =~ /\[\]$/
-          @array = true
-          super(name, default, cast_type, sql_type[0..sql_type.length - 3], null)
```

Comment:
This lines has been introduced from https://github.com/rails/rails/pull/7547:

https://github.com/rails/rails/pull/7547/files#diff-cc31fc1dd1e78db64638200f710b2f59R51

However, I think that there was no reason to remove `[]`.